src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/MethodNameCheck.java

```diff
@@ -83,6 +83,17 @@
  * <pre>
  * &lt;module name="MethodName"/&gt;
  * </pre>
+ * <p>Code Example:</p>
+ * <pre>
+ * class MyClass {
+ *   public void firstMethod1() {} // OK
+ *   protected void secondMethod2() {} // OK
```

Comment:
remove 2 from end. It is obvious from the 1st example that you could use number with methodnaming, so rest should be more cleaner in look.

Reply:
done.

Reply:
resolved